src/app/common/focus-tab.ts

```diff
@@ -0,0 +1,9 @@
+import { analytics } from '@shared/utils/analytics';
+
+export function focusInitatingTab(tabId: number | null) {
+  void analytics.track('user_clicked_requested_by_link', { endpoint: 'getAddresses' });
+  chrome.tabs.update(tabId ?? 0, { active: true }, tab => {
+    if (!tab) return;
+    chrome.windows.update(tab.windowId, { focused: true });
+  });
+}
```

Comment:
I've refactored this out in my PR, which I'll merge shortly. We shoudln't fire this analytics event directly in the handler for example.